epam-assets/theme/theme_electric.scss

```diff
@@ -444,10 +476,11 @@
         --uui-badge-caption-hover: var(--uui-neutral-80);
     }
 
-    .uui-badge.uui-color-neutral {
+    .uui-badge.uui-color-neutral,
+    .uui-badge.uui-color-night300 {
```

Comment:
we don't need to have alias 'night300' for the 'neutral' color

Reply:
fixed